userspace/engine/yaml_helper.h

```diff
@@ -89,7 +84,37 @@ class yaml_helper
 	inline static const std::string configs_key = "config_files";
 	inline static const std::string validation_ok = "ok";
 	inline static const std::string validation_failed = "failed";
-	inline static const std::string validation_none = "schema not provided";
+	inline static const std::string validation_none = "none";
```

Comment:
`none` better reflect the fact that no validation took place, without exposing more info than needed.